CMakeLists.txt

```
@@ -30,7 +35,8 @@ if (OPENMP_FOUND)
     set (CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} ${OpenMP_EXE_LINKER_FLAGS}")
 endif()
 endif()
-find_package(Boost EXACT COMPONENTS system thread filesystem regex timer REQUIRED)
+find_package(Boost COMPONENTS system thread filesystem regex timer REQUIRED)
+INCLUDE_DIRECTORIES( ${Boost_INCLUDE_DIR} )
```

```suggestion
include_directories( ${Boost_INCLUDE_DIR} )
```
Minor change, but I'd keep all the cmake functions lowercase.